sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpPlugin.java

```diff
@@ -72,40 +74,62 @@ public void define(Context context) {
       context.addExtensions(
         PropertyDefinition.builder(PHPUNIT_TESTS_REPORT_PATH_KEY)
           .name("Unit Test Report")
-          .description("Path to the PHPUnit unit test execution report file. The path may be either absolute or relative to the project base directory.")
+          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, "", "unit test execution"))
           .onQualifiers(Qualifiers.MODULE, Qualifiers.PROJECT)
           .category(PHP_CATEGORY)
           .subCategory(PHPUNIT_SUBCATEGORY)
           .build(),
 
         PropertyDefinition.builder(PHPUNIT_COVERAGE_REPORT_PATH_KEY)
           .name("Coverage Report")
-          .description("Path to the PHPUnit code coverage report file. The path may be either absolute or relative to the project base directory.")
+          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(context), "code coverage"))
           .onQualifiers(Qualifiers.MODULE, Qualifiers.PROJECT)
           .category(PHP_CATEGORY)
           .subCategory(PHPUNIT_SUBCATEGORY)
           .build(),
 
         PropertyDefinition.builder(PHPUNIT_IT_COVERAGE_REPORT_PATH_KEY)
           .name("IT Coverage Report")
-          .description("Path to the PHPUnit integration test code coverage report file. The path may be either absolute or relative to the project base directory.")
+          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(context), "integration test code coverage"))
           .onQualifiers(Qualifiers.MODULE, Qualifiers.PROJECT)
           .category(PHP_CATEGORY)
           .subCategory(PHPUNIT_SUBCATEGORY)
           .build(),
 
         PropertyDefinition.builder(PHPUNIT_OVERALL_COVERAGE_REPORT_PATH_KEY)
           .name("Overall Coverage Report")
-          .description("Path to the PHPUnit overall code coverage report file. The path may be either absolute or relative to the project base directory.")
+          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(context), "overall code coverage"))
           .onQualifiers(Qualifiers.MODULE, Qualifiers.PROJECT)
           .category(PHP_CATEGORY)
           .subCategory(PHPUNIT_SUBCATEGORY)
           .build());
     }
+    if (versionSupportsMultiPathCoverageReport(context)) {
+      context.addExtension(PropertyDefinition.builder(PHPUNIT_COVERAGE_REPORT_PATHS_KEY)
+        .name("Coverage Reports")
+        .description("Comma-separated list of PHPUnit code coverage report files. Each path can be either absolute or relative.")
+        .onQualifiers(Qualifiers.MODULE, Qualifiers.PROJECT)
+        .category(PHP_CATEGORY)
+        .subCategory(PHPUNIT_SUBCATEGORY)
+        .build());
+    }
+
+  }
+
+  private String deprecationMessage(Context context) {
+    return versionSupportsMultiPathCoverageReport(context) ? "DEPRECATED: use " + PHPUNIT_COVERAGE_REPORT_PATHS_KEY + ". " : "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS864)